tests/testthat/test-row_count.R

```diff
@@ -0,0 +1,39 @@
+test_that("row_count", {
+  d_mn <- data.frame(
+    c1 = c(1, 2, NA, 4),
+    c2 = c(NA, 2, NA, 5),
+    c3 = c(NA, 4, NA, NA),
+    c4 = c(2, 3, 7, 8)
+  )
+  expect_identical(row_count(d_mn, count = 2), c(1, 2, 0, 0))
+  expect_identical(row_count(d_mn, count = NA), c(2, 0, 3, 1))
```

Comment:
can you add a test with `count = NULL` and one with a 0-row dataframe (just to check we don't accidentally create a row in this case)?

Reply:
count = NULL is in line 23 (`count` is checked first, before values of `select`)